meta-mender-tegra/recipes-mender/tegra-state-scripts/files/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do not think it is necessary to provide your own copy of the Apache 2.0 License, and instead you can use the following in your recipe:

```
LIC_FILE_CHECKSUM = "${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
```

Reply:
OK agreed, will do, I was copying the example at https://github.com/mendersoftware/meta-mender/tree/master/meta-mender-demo/recipes-mender/example-state-scripts/files FYI.